### PR TITLE
docs: document how to choose the agent type

### PR DESCRIPTION
**Pull request by kuisathaverat**
Adds instructions about how to select the Agent Type

related to https://github.com/elastic/opbeans-java/pull/107

### CODE REVIEW

**Review by jlvoiseux — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,3 +59,12 @@ Another possible way is to create a different property file like application-cus
 
 The application has a built-in bug that you can trigger by
 navigating to the path `/is-it-coffee-time`.
+
+## Agent mode (elasticapm/opentelemetry)
+
+The Opbeans Java can use the APM Agent Java or the OpenTelemetry java implementation
+in order to choose one of other you have to set the environment variable APM_AGENT_TYPE
+to `elasticapm` when you use the APM Agent Java and `opentelemetry` to use OpenTelemetry java implementation.
```
Comment (SylvainJuge) [inline README.md:67]: Now there is also the `none` value that will make Opbeans run without any instrumentation agent.

In this case it means that if OTel environment variables are properly et the OTel SDK will allow the application to send spans created through manual instrumentation.

File: README.md [hunk 2]
```diff
@@ -59,3 +59,12 @@ Another possible way is to create a different property file like application-cus
...
+
+see the [docker-compose](./docker-compose.yml) file for more details about the nevironment variables.
```
Comment (cachedout) [inline README.md:69]:
```suggestion
see the [docker-compose](./docker-compose.yml) file for more details about the environment variables.
```

**Review by cachedout — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -59,3 +59,13 @@ Another possible way is to create a different property file like application-cus
 
 The application has a built-in bug that you can trigger by
 navigating to the path `/is-it-coffee-time`.
+
+## Agent mode (elasticapm/opentelemetry)
+
+The Opbeans Java can use the APM Agent Java or the OpenTelemetry java implementation
+in order to choose one of other you have to set the environment variable APM_AGENT_TYPE
+to `elasticapm` when you use the APM Agent Java and `opentelemetry` to use OpenTelemetry java implementation.
+Finally, there is also the none value that will make Opbeans run without any instrumentation agent.
```
Comment (kuisathaverat) [inline README.md:68]:
```suggestion
Finally, there is also the `none` value that will make Opbeans run without any instrumentation agent.
```

File: README.md [hunk 4]
```diff
@@ -59,3 +59,13 @@ Another possible way is to create a different property file like application-cus
...
 navigating to the path `/is-it-coffee-time`.
+
+## Agent mode (elasticapm/opentelemetry)
+
+The Opbeans Java can use the APM Agent Java or the OpenTelemetry java implementation
+in order to choose one of other you have to set the environment variable APM_AGENT_TYPE
+to `elasticapm` when you use the APM Agent Java and `opentelemetry` to use OpenTelemetry java implementation.
+Finally, there is also the `none` value that will make Opbeans run without any instrumentation agent.
+
+see the [docker-compose](./docker-compose.yml) file for more details about the environment variables.
```
Comment (kuisathaverat) [inline README.md:70]:
```suggestion
When OpenTelemetry agent is enabled, the OpenTelemetry environment variables should be properly set to allow the application to send spans created through manual instrumentation.
see the [docker-compose](./docker-compose.yml) file for more details about the environment variables, and [OpenTelemetry SDK Autoconfigure](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md) for more details.
```

**Pull request merged**